Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -88,11 +89,11 @@ void Jit64::psq_stXX(UGeckoInstruction inst)
 		}
 
 		BitSet32 registersInUse = CallerSavedRegistersInUse();
-		if (update && js.memcheck)
+		if (update && storeAddress)
```

Comment:
This would become
if (update && ((update && js.memcheck) || !SConfig::GetInstance().m_LocalCoreStartupParameter.bFastmem))

Why are we checking update twice in the same if statement?